include/jemalloc/internal/tcache_structs.h

```diff
@@ -55,6 +55,8 @@ struct tcache_slow_s {
 
 struct tcache_s {
 	tcache_slow_t	*tcache_slow;
+	unsigned	tcache_nhbins;
```

Comment:
As discussed with @interwq offline, moving nhbins and tcache_max into `tcache_s` so that manually created tcaches can be well supported too.